### DISCUSSION TITLE
A discord server for the project

### DISCUSSION THREAD

**Discussion by HassanJbara** [bcollazo/catanatron#272] [Ideas]
Hi, I came across this project while doing some RL research and I found it really cool! I hope to contribute to it in the future. Seeing that you are still working on this, did you consider creating a discord server or some sort of group chat for faster cooperation and sharing of ideas?

**Comment by bcollazo** [comment]
Hey HassanJbara, great to hear you found the project cool. Hopefully the gym interface can be of help for your research!

Haven't considered creating a discord server yet, but I've collaborated via email with several individuals. The idea is good, I just rather wait until it gets to a point where we reach a critical-mass and/or there's a decent amount of correspondence. Feel free to email me or use the issues / discussions here in the meantime!

**Comment by zarns** [comment]
@bcollazo, looks like there's a Discord server now! :rocket:
The link in the README is expired... I'd love to join!

**Reply by bcollazo** [reply]
Hey @zarns, thanks for noticing! There, I think I fixed it. Let me know if it still doesn't work! 🚀

**Reply by zarns** [reply]
I'm in! Thanks
